epoxy-processor/src/main/java/com/airbnb/epoxy/QualifiedId.java

```diff
@@ -0,0 +1,48 @@
+package com.airbnb.epoxy;
+
+/**
+ * Created by ilyazorin on 12/03/2017.
```

Comment:
Remove the created by javadoc please. We like relying on git blame instead of code comments since those can become out of date.